dev/NumberBox/NumberBox.cpp

```diff
@@ -298,6 +305,22 @@ void NumberBox::UpdateValueToText()
     }
 }
 
+void NumberBox::OnHeaderPropertyChanged(const winrt::DependencyPropertyChangedEventArgs& args)
+{
+    // To enable lightweight styling, collapse header presenter if there is no header specified
+    if (const auto headerPresenter = m_headerPresenter.get())
+    {
+        if (const auto header = Header())
```

Comment:
>header [](start = 23, length = 6)

Does setting NumberBox's header in markup like so `Header = ""` cause the header to be set to nullptr or be set to empty string? if its the latter should we also collapse the header presenter in that case?

Reply:
@chingucoding do you have an answer to this before we check it in?

Reply:
Currently, it would render the presenter with empty text. I think it should be collapsed in that case. I will update the PR.

Reply:
Update: Fixed and test adjusted.